test/index.test.js

```diff
@@ -203,6 +203,10 @@ describe('Index Tests', () => {
             ...input,
           };
           await main(params);
+          if (!input.observation && queues[name]) {
+            // eslint-disable-next-line no-param-reassign
+            queues[name].forEach(({ record }) => delete record.eventTime);
```

Comment:
For manual invocations, eventTime will be the current time, so we need to eliminate it from the comparison.